jwtlake/pokemon-go-helper
# Make login form accessible

- **Problem:** plain login form doesn't submit with `Return` key.
- **Solution:** Use a `form` element and ensure the login button has `type="submit"`
- **Test:**
  1. Ensure your bundle is built (`npm run webpack`)
  2. Start server (`npm start`)
  3. Open localhost:3000 in your browser
  4. Fill out form with username: test, password: test
  5. Press `Return` key
  6. 🎉 Success!
